lib/app/views/device_error_screen.dart

```diff
@@ -48,7 +48,7 @@ class DeviceErrorScreen extends ConsumerWidget {
                     context, 'Elevating permissions...',
                     duration: const Duration(seconds: 30));
                 try {
-                  if (await ref.read(rpcProvider).elevate()) {
+                  if (await ref.read(rpcProvider).value!.elevate()) {
```

Comment:
Is there a reason for having `value!` here instead of `requireValue`?

Reply:
There isn't, good catch!